WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.kt

```diff
@@ -385,9 +383,6 @@ class NotificationsListFragment : Fragment(R.layout.notifications_list_fragment)
             }
             val detailIntent = getOpenNoteIntent(activity, noteId)
             detailIntent.putExtra(NOTE_INSTANT_REPLY_EXTRA, shouldShowKeyboard)
-            if (!TextUtils.isEmpty(replyText)) {
-                detailIntent.putExtra(NOTE_PREFILLED_REPLY_EXTRA, replyText)
```

Comment:
I think we can also remove the `NOTE_PREFILLED_REPLY_EXTRA` declaration above.

Reply:
done